.github/workflows/notify.yaml

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
:question: Is this is a left over debug file, it doesn't look like this does anything.

Reply:
Yeah, will remove